vars/sharedInfrastructurePipeline.groovy

```diff
@@ -29,8 +29,8 @@ ______                              _           _ _
 ================================================================================
 '''
 
-  try {
-    node {
+  node {
```

Comment:
should really just not use this pipeline instead...

Reply:
https://github.com/hmcts/bulk-scan-shared-infrastructure/pull/144